src/Importer.php

```diff
@@ -19,12 +19,11 @@
 use Maatwebsite\Excel\Concerns\WithValidation;
 use SimonHamp\LaravelNovaCsvImport\Concerns\HasModifiers;
 
-class Importer implements ToModel, WithValidation, WithHeadingRow, WithMapping, WithBatchInserts, WithChunkReading,
-                            SkipsOnFailure, SkipsOnError, SkipsEmptyRows
+class Importer implements ToModel, WithValidation, WithHeadingRow, WithMapping, WithBatchInserts, WithChunkReading, SkipsOnFailure, SkipsOnError, SkipsEmptyRows
 {
     use Importable, SkipsFailures, SkipsErrors, HasModifiers;
 
-    /** @var Resource */
+    /** @var resource */
```

Comment:
I disagree with this one. It should be referencing the class `Resource`?

Reply:
ill look into why, i also noticed that latest pint requires php 8.1 :/ not sure how you feel about that.
although Laravel 10 requires it, nova does not. depends on your userbase.

Reply:
i think you can remove it and use
```php
protected Resource $resource
```
otherwise i believe it should be
```php
 /** @var Resource $resource */

 ```

Reply:
actually you can move all properties to the constructor :) . But thats a whole other PR

since https://stitcher.io/blog/constructor-promotion-in-php-8

but ill stick to the comment since its a smaller change

Reply:
hmm pint just reverts it , saying [phpdoc_types](https://cs.symfony.com/doc/rules/phpdoc/phpdoc_types.html), [unary_operator_spaces](https://cs.symfony.com/doc/rules/operator/unary_operator_spaces.html), [not_operator_with_successor_space](https://cs.symfony.com/doc/rules/operator/not_operator_with_successor_space.html)
```sh
vendor/bin/pint -v

  .........✓.......

  ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── Laravel
    FIXED   ...................................................................................................................................... 17 files, 1 style issue fixed
  ✓ src/Importer.php                                                                                      phpdoc_types, unary_operator_spaces, not_operator_with_successor_space
  @@ -23,7 +23,7 @@
   {
       use Importable, SkipsFailures, SkipsErrors, HasModifiers;

  -    /** @var Resource */
  +    /** @var resource */
       protected $resource;

       protected $attribute_map = [];
```